code/modules/hydroponics/grown/citrus.dm

```diff
@@ -32,7 +32,69 @@
 	icon_state = "lime"
 	filling_color = "#00FF00"
 	juice_results = list(/datum/reagent/consumable/limejuice = 0)
+//Lye-mes
+/obj/item/seeds/lime/lyeme
+	name = "pack of lye-me seeds"
+	desc = "These are very clean."
+	icon_state = "seed-lyeme"
+	species = "lyeme"
+	plantname = "Lye-me Tree"
+	product = /obj/item/reagent_containers/food/snacks/grown/citrus/lyeme
+	lifespan = 55
+	endurance = 50
+	yield = 4
+	potency = 15
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	genes = list(/datum/plant_gene/trait/slip,/datum/plant_gene/trait/repeated_harvest)
+	reagents_add = list(/datum/reagent/lye = 0.15, /datum/reagent/consumable/nutriment = 0.05)
+	rarity = 40
 
+/obj/item/reagent_containers/food/snacks/grown/citrus/lyeme
+	seed = /obj/item/seeds/lime/lyeme
+	name = "lye-me"
+	desc = "These could defeat the most robust of stains."
+	icon_state = "lyeme"
+	filling_color = "#00FF00"
+	var/cleanspeed = 50
+/obj/item/reagent_containers/food/snacks/grown/citrus/lyeme/add_juice()
+	..()
+	cleanspeed =round(50-(seed.potency/2))
```

Comment:
change this to be
cleanspeed =round(**60**-(seed.potency/2))

We don't want lye-mes with a cleanspeed of 0, right?

Reply:
good point